dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/ingest/IngestException.java

```diff
@@ -0,0 +1,43 @@
+package edu.harvard.iq.dataverse.ingest;
+
+import edu.harvard.iq.dataverse.persistence.datafile.ingest.IngestError;
+
+import javax.ejb.ApplicationException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@ApplicationException(rollback = true)
+public class IngestException extends RuntimeException {
+
+    private IngestError errorKey;
+    private List<String> bundleArguments = new ArrayList<>();
+
+    // -------------------- CONSTRUCTORS --------------------
+
+    public IngestException(IngestError errorKey) {
+        super("");
```

Comment:
If you pass empty string to the `super` constructor, when this exception gets logged it will have an empty message. That's rather undesirable. Let's replace it with some generic message containing the error key and arguments instead.